timesketch/lib/analyzers/feature_extraction.py

```diff
@@ -229,7 +266,46 @@ def extract_feature(self, name, config):
                 continue
 
             event_counter += 1
-            event.add_attributes({store_as: result[0]})
+            store_type = event.source.get(store_as)
+            if store_type and not overwrite_store_as:
+                continue
+            if isinstance(store_type, six.text_type):
+                store_type_list = False
+            elif isinstance(store_type, (list, tuple)):
+                store_type_list = True
+            if store_type_list:
+                if keep_multimatch:
+                    if store_type and overwrite_and_merge_store_as:
+                        result_list = list(set(store_type) | set(result))
+                        event.add_attributes({store_as: result_list})
+                    else:
+                        event.add_attributes({store_as: result})
+                else:
+                    if overwrite_and_merge_store_as and \
```

Comment:
again, don't use \ to divide between lines

here if you use the function you can a) save a lot of indentation and b) return early to avoid all this